bin/codebasin

```diff
@@ -220,6 +242,32 @@ def main():
             filtered_platforms=filtered_platforms,
         )
 
+    # Load the analysis file if it exists.
+    if args.analysis_file is not None:
+        path = os.path.realpath(args.analysis_file)
+        if os.path.exists(path):
+            if not os.path.splitext(path)[1] == ".toml":
+                raise RuntimeError(f"Analysis file {p} must end in .toml.")
+
+        with util.safe_open_read_nofollow(path, "rb") as f:
+            try:
+                analysis_toml = util._load_toml(f, "analysis")
```

Comment:
I don't seem to resolve this function in `util`... Can you confirm?

Reply:
Isn't it here:
https://github.com/intel/code-base-investigator/blob/bf5b31e88b6f3b9177dc832aa147f267aae29ef4/codebasin/util.py#L265

Reply:
Huh weird, maybe Github lead me to the wrong commit...